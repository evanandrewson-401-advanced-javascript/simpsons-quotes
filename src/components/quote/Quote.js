import React from 'react';
import PropTypes from 'prop-types';

const Quote = ({ quote, characterImage, characterName }) => {
  return (
    <>
      <p>{quote}</p>
      <img src={characterImage} alt={characterName} />
      <p>{characterName}</p>
    </>
  );
};

Quote.propTypes = {
  quote: PropTypes.string.isRequired,
  characterName: PropTypes.string.isRequired,
  characterImage: PropTypes.string.isRequired,
};

export default Quote;
