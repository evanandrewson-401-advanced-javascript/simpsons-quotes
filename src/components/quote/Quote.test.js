import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote component', () => {
  it('renders Quote component', () => {
    const wrapper = shallow(<Quote quote={''} characterImage={''} characterName={''}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
