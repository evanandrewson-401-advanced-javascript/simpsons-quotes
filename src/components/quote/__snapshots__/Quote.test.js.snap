// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders Quote component 1`] = `
<Fragment>
  <p />
  <img
    alt=""
    src=""
  />
  <p />
</Fragment>
`;
